node/src/components/storage.rs

```diff
@@ -317,9 +317,29 @@ enum GetRequestError {
     GossipedAddressNotGettable,
 }
 
-/// The storage component.
+/// Storage component.
 #[derive(DataSize, Debug)]
 pub struct Storage {
+    storage: StorageInner,
+}
+
+impl Storage {
+    pub(crate) fn read_switch_block_header_by_era_id(
+        &self,
+        switch_block_era_id: EraId,
+    ) -> Result<Option<BlockHeader>, FatalStorageError> {
+        self.storage
+            .read_switch_block_header_by_era_id(switch_block_era_id)
```

Comment:
Why is this the only method that gets exposed on `Storage`?

Reply:
Any method that is exposed on `Storage` is there because external tooling (think `utils`) needs it, or some other functionality access the component directly (e.g. in reactor creation).